docs/onboarding/azure-devops-pipelines.md

```diff
@@ -819,3 +820,141 @@ In order to configure audit stream for Azure Monitor, identify the following inf
+    
+    az account management-group show --name $TENANT_ROOT_GROUP_ID -e -r | jq 'recurse(.children[]?) |= del(select(.type == "/subscriptions")) | del(..|nulls) | recurse(.children[]?) |= {"id":.name, "name":.displayName, "children": (.children // [] )}'
+    ```
+
+    This command will:
+
+    * Use Azure CLI to retrieve the management group structure recursively from `Tenant Root Group`.  Response from Azure CLI will include subscriptions.
+    * Use `jq` to
+      * Remove all objects that are of type: `/subscription`
+      * Removing the subscription objects leaves `null` in place in arrays so we then delete the nulls references
+      * Recurse through the intermediate json and select `name`, `id` and `children` (if children is null, then mark it as an empty array `[]`).
+
+    Output:
+
+    ```json
+    {
+      "id": "<< ENTER YOUR TENANT ROOT GROUP ID >>",
+      "name": "Tenant Root Group",
+      "children": [
+        {
+          "id": "pubsec",
+          "name": "Azure Landing Zones for Canadian Public Sector",
+          "children": [
+            {
+              "id": "pubsecSandbox",
+              "name": "Sandbox",
+              "children": []
+            },
+            {
+              "id": "pubsecPlatform",
+              "name": "Platform",
+              "children": [
+                {
+                  "id": "pubsecPlatformIdentity",
+                  "name": "Identity",
+                  "children": []
+                },
+                {
+                  "id": "pubsecPlatformManagement",
+                  "name": "Management",
+                  "children": []
+                },
+                {
+                  "id": "pubsecPlatformConnectivity",
+                  "name": "Connectivity",
+                  "children": []
+                }
+              ]
+            },
+            {
+              "id": "pubsecLandingZones",
+              "name": "Landing Zones",
+              "children": [
+                {
+                  "id": "pubsecLandingZonesQA",
+                  "name": "QA",
+                  "children": []
+                },
+                {
+                  "id": "pubsecLandingZonesDevTest",
+                  "name": "Dev/Test",
+                  "children": []
+                },
+                {
+                  "id": "pubsecLandingZonesProd",
+                  "name": "Production",
+                  "children": []
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+    ```
+
+3. Update `./config/variables/<devops-org-name>-<branch-name>.yml`
+
+    * Comment or delete the following two variables.  They will be queried from the management group hierarchy and set by Azure DevOps for pipeline execution:
+      * var-parentManagementGroupId
+      * var-topLevelManagementGroupName
+    * Add new variable `var-managementgroup-hierarchy` with the JSON output from above.
```

Comment:
Advise to be careful with indentation -> YAML is indentation sensitive, and failure to observe this may cause errors that are difficult to diagnose and correct.